### PR TITLE
generated bookings model

### PR DESCRIPTION
**Pull request by LHB410**
Hello! I generated the bookings model if someone could check it that'd be great!

### CODE REVIEW

File: db/migrate/20220125134340_create_bookings.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateBookings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bookings do |t|
+
```
Comment (Laeti-dev) [inline db/migrate/20220125134340_create_bookings.rb]: When we add a booking shouldn't we have user:references & lesson:references ?

Comment (LHB410) [reply db/migrate/20220125134340_create_bookings.rb]: ok will fix this then re commit

**Review by LHB410 — COMMENTED** [review]
have added foreign keys

File: db/migrate/20220125134340_create_bookings.rb (hunk 1 above)
Comment (LHB410) [reply db/migrate/20220125134340_create_bookings.rb]: fixed!!

**Review by Laeti-dev — APPROVED** [review]

**Pull request merged**